openupgradelib/openupgrade.py

```diff
@@ -507,16 +507,24 @@ def rename_columns(cr, column_spec):
     """
     for table in column_spec.keys():
         for (old, new) in column_spec[table]:
+            rename_index = True
+
             if new is None:
                 new = get_legacy_name(old)
+                rename_index = False
+
             logger.info("table %s, column %s: renaming to %s",
                         table, old, new)
             cr.execute(
                 'ALTER TABLE "%s" RENAME "%s" TO "%s"' % (table, old, new,))
-            cr.execute(
-                'ALTER INDEX IF EXISTS "%s_%s_index" RENAME TO "%s_%s_index"' %
-                (table, old, table, new)
-            )
+
+            if rename_index:
```

Comment:
Why not simply `if new is not None:` here?

Reply:
Because at that point `new` can never be `None` due to the code on line 513 where `new` get value of the legacy name or am I missing something?

Reply:
Good point! Let's see the general discussion about dropping or not.